Sources/SuggestionRow.swift

```diff
@@ -17,6 +17,7 @@ open class _SuggestionRow<Cell: CellType>: FieldRow<Cell> where Cell: BaseCell,
 //SuggestionCell<T>
     public var filterFunction: ((String) -> [Cell.Value])!
     public var asyncFilterFunction: ((String, (@escaping ([Cell.Value]) -> Void)) -> Void)?
+    public var maxSuggestionHeight: CGFloat? // How many row results to show at a time in the view
```

Comment:
As this is the count of rows, I would rename it to something more like `maxSuggestionRows` or `maxSuggestionCount`. Also I think it should be an `Int` to avoid confusion and strange behaviour as someone could specify a float value.

Reply:
I originally had it as an Int but was not able to do 44 * maxSuggestionHeight as an Int due to a compile error that it needed to be a CGFloat. Let me know if you think there is a better way to handle that.

I also originally had it named maxSuggestionCount. However, I noticed that all the rows still showed up if the number was 5, you just had to scroll to see them. I renamed it to height because that was the number needed to see all results at once. I can use maxSuggestionRows if that makes more sense.

Reply:
I pushed the requested changes